### PR TITLE
[wip] Weekly News, 6 May 2016

### PR DESCRIPTION
**Pull request by smashwilson**
@rgbkrk: I've never actually done one of these before, so I'd appreciate another set of eyes :bow:

@everett-toews: I've omitted the improved error messages announcement from this week's post since it's not shipping until Monday, but [it's still in the branch history](https://github.com/getcarina/getcarina.com/commit/6ab30da6c027da66c365ada059536a586472bbfa#diff-77d44fad798edb1b8458d3c0f1bc0591R24) if you'd like to nab it and edit for next week's.

Fixes rackerlabs/dxeng#115.

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-e5939d263c/)

File: _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Server:
+ Version:      swarm/1.2.0
+ API version:  1.22
+ Go version:   go1.5.4
+ Git commit:   a6c1f14
+ Built:        Wed Apr 13 05:58:31 UTC 2016
+ OS/Arch:      linux/amd64
+```
+
+`dvm` has been around for a while, and thanks to the continued efforts of Carolyn Van Slyck, it's now reached a level of maturity that makes us confident in recommending it to our users. Many of our tutorials that previously referenced manual Docker installations now use `dvm` instead.
```
Comment (smashwilson) [inline _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md]: Hmm, "recommending" isn't the right word here, we've done that all along. How else can I say "it's the default for our docs now"?

File: _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+authorIsRacker: true
+---
+
+This week's highlights include the use of DVM in our tutorials and Carina representation at OSCON. But first, a little Shakespeare.
+
+<!-- more -->
+
+### Shakespeare
+
+Did you know that there's a esoteric programming language inspired by the works of William Shakespeare? Declare variables in a Dramatis Personæ, implement control flow with stage actions, and decrement counters with insults. Jamie Hannaford helped us celebrate the 400th anniversary of Shakespeare's death by [running some Shakespeare code on Carina]({{ site.baseurl }}/blog/celebrating-shakespeare/).
```
Comment (rgbkrk) [inline _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md]: > Though this be madness, yet there is method in 't.

File: _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md (hunk 1 above)
Comment (rgbkrk) [inline _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md]: Seems ok with me. The final sentence here is good as well.

**Comment by rgbkrk** [comment]
Looking good to me.

File: _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ Git commit:   a6c1f14
+ Built:        Wed Apr 13 05:58:31 UTC 2016
+ OS/Arch:      linux/amd64
+```
+
+`dvm` has been around for a while, and thanks to the continued efforts of Carolyn Van Slyck, it's now reached a level of maturity that makes us confident in recommending it to our users. Many of our tutorials that previously referenced manual Docker installations now use `dvm` instead.
+
+### Upcoming appearances
+
+Next week O'Reilly's annual open source convention, [OSCON](http://conferences.oreilly.com/oscon/open-source-us), will take place in Austin, Texas. Catch members of the Carina team at any of these events:
```
Comment (everett-toews) [inline _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md]: OSCON is the week after next.

File: _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+* Tuesday at 9:05am, as part of Open Container Day, our own Everett Toews is hosting an ["Intro to Docker Swarm"](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/50961) tutorial.
+* Wednesday at 11:05am, Everett will be presenting ["Effective Docker Swarm"](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/51213) to share additional insights on developing and deploying real-world applications on Docker Swarm.
+* Wednesday at 2:40pm, Carolyn Van Slyck, Nick Silkey, and yours truly Ash Wilson will be presenting ["Think outside the container"](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/51253), an exploration of unconventional use-cases for Docker containers.
```
Comment (everett-toews) [inline _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md]: People can get into the OSCON Expo Hall and Open Container Day for free. Some suggestions (from an O'Reilly email)

* Check out #OSCON in Austin for FREE. Register for an expo hall pass w/ code SPEXPO. See everything included: http://oreil.ly/1T0euIj
* Get 4 days of #OSCON May 16-19 free w/ Expo Plus pass (use code SPEXPO). Access to events, special programs + more http://oreil.ly/1T0euIj
* Free Expo Plus pass for #OSCON with code SPEXPO. Includes events, All the Bots Day, Open Container Day + more http://oreil.ly/1T0euIj

File: _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+This week's highlights include the use of DVM in our tutorials and Carina representation at OSCON. But first, a little Shakespeare.
+
+<!-- more -->
+
+### Shakespeare
+
+Did you know that there's a esoteric programming language inspired by the works of William Shakespeare? Declare variables in a Dramatis Personæ, implement control flow with stage actions, and decrement counters with insults. Jamie Hannaford helped us celebrate the 400th anniversary of Shakespeare's death by [running some Shakespeare code on Carina]({{ site.baseurl }}/blog/celebrating-shakespeare/).
+
```
Comment (everett-toews) [inline _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md]: How about a Shakespeare image here?

**Comment by rgbkrk** [comment]
I just dug through previous news weeklies and noticed that the `fish` support wasn't listed. You're free to add it here, though support in `carina` is waiting on https://github.com/getcarina/carina/pull/108

File: _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md (hunk 3 above)
Comment (smashwilson) [inline _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md]: Er, oops. Should I still include it here?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-114064d5ec/)

File: _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md (hunk 3 above)
Comment (everett-toews) [inline _posts/2016-05-05-weekly-news-shakespeare-dvm-oscon.md]: Might as well include it. I'll include it again next week too.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-e160f40efb/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-b8951bcc7c/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-fb71ec2f23/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-63c2852edf/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-a713de773b/)

**Comment by kmsholcomb** [comment]
Hi @smashwilson. I am going to leave some copy edit comments in the file. I will let you know when I am done. Thanks!

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+comments: true
+author: Ash Wilson <ash.wilson@rackspace.com>
+published: true
+categories:
+- News
+- Shakespeare
+- Conferences
+- OSCON
+- dvm
+authorIsRacker: true
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: Other weekly posts have a slightly different set of metadata. Compare with any of the recently published ones. One thing you seem to be missing is the “excerpt” field.

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+This week's highlights include container logs in the Carina web UI, the use of `dvm` in our tutorials, and Carina representation at OSCON. But first, a little Shakespeare.
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: ... in the Carina Control Panel, the use of Docker Version Manager (dvm) in our tutorials, ...

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+<!-- more -->
+
+### Shakespeare
+
+Did you know that there's a esoteric programming language inspired by the works of William Shakespeare? Declare variables in a Dramatis Personæ, implement control flow with stage actions, and decrement counters with insults. Jamie Hannaford helped us celebrate the 400th anniversary of Shakespeare's death by [running some Shakespeare code on Carina]({{ site.baseurl }}/blog/celebrating-shakespeare/).
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: ... there's an esoteric ...

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+![Shakespeare nods approvingly at your code]({% asset_path 2016-05-06-weekly-news-shakespeare-logs-oscon/Shakespeare.jpg %})
+
+### Container logs in the Carina web UI
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: in the Carina Control Panel

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+Watch your containers' logs in real time, right from the [Carina control panel](https://app.getcarina.com/app/login)! Click on a cluster name, scroll to the "Containers" section, and click on the gear icon next to a container to reveal the option:
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: ... right from the [Carina Control Panel](https://app.getcarina.com/app/login)! Click a cluster name, scroll to the **Containers** section, and click the gear icon next to a container to reveal the option:

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+!["container logs" option]({% asset_path 2016-05-06-weekly-news-shakespeare-logs-oscon/container-logs-option.jpg %})
+
+Clicking the link will launch the log viewer:
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: Click the link to start the log viewer:

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+![log viewer]({% asset_path 2016-05-06-weekly-news-shakespeare-logs-oscon/log-viewer.jpg %})
+
+Add multiple container names to watch output from many containers at once, like `docker-compose`:
+
+![M-m-m-m-m-multilogs]({% asset_path 2016-05-06-weekly-news-shakespeare-logs-oscon/m-m-m-m-multi-logs.jpg %})
+
+### Fish support
+
+Credential .zip files contain a `docker.fish` file. If you use the [fish shell](https://fishshell.com/), source this file instead of `docker.env` to gain access to your cluster.
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: Credentials zip files contain ...

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+🐟 🐟 🐟
+
+Support for `docker.fish` in the [`carina` command-line client](https://github.com/getcarina/carina) is coming soon.
+
+![M-m-m-m-m-multilogs]({% asset_path 2016-05-06-weekly-news-shakespeare-logs-oscon/fishy-fish.jpg %})
+
+### Docker version manager
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: Docker Version Manager

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+
+If you've ever juggled multiple Carina clusters that all use different Docker versions, then you've met the dreaded ["client and server don't have the same version" error]({{ site.baseurl }}/docs/troubleshooting/common-problems/#error-response-from-daemon-client-and-server-don-39-t-have-same-version). Our solution to keep everything synchronized is the [Docker version manager, `dvm`](https://github.com/getcarina/dvm).
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: ... Carina clusters that use different ...

... synchronized is the [Docker Version Manager (dvm)](https://github.com/getcarina/dvm).

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+Server:
+ Version:      swarm/1.2.0
+ API version:  1.22
+ Go version:   go1.5.4
+ Git commit:   a6c1f14
+ Built:        Wed Apr 13 05:58:31 UTC 2016
+ OS/Arch:      linux/amd64
+```
+
+`dvm` has been around for a while, and thanks to the continued efforts of Carolyn Van Slyck, it's now reached a level of maturity that makes us confident in recommending it to our users. Many of our tutorials that previously referenced manual Docker installations now use `dvm` instead.
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: Although dvm has been around for a while, thanks ...

... Van Slyck, it has now ...

... now use dvm instead.

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+
+### Upcoming appearances
+
+The week after next O'Reilly's annual open source convention, [OSCON](http://conferences.oreilly.com/oscon/open-source-us), will take place in Austin, Texas. Catch members of the Carina team at any of these events:
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: The week of May 16, O'Reilly's ...

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+
+* Tuesday at 9:05am, as part of Open Container Day, our own Everett Toews is hosting an ["Intro to Docker Swarm"](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/50961) tutorial.
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: - Tuesday at 9:05 a.m., ... an [Intro to Docker Swarm](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/50961) tutorial.

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+* Wednesday at 11:05am, Everett will be presenting ["Effective Docker Swarm"](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/51213) to share additional insights on developing and deploying real-world applications on Docker Swarm.
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: - Wednesday at 11:05 a.m., Everett will be presenting [Effective Docker Swarm](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/51213)  to share additional insights about developing ...

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+* Wednesday at 2:40pm, Carolyn Van Slyck, Nick Silkey, and yours truly Ash Wilson will be presenting ["Think outside the container"](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/51253), an exploration of unconventional use-cases for Docker containers.
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: - Wednesday at 2:40 p.m., Carolyn Van Slyck, Nick Silkey, and yours truly Ash Wilson will be presenting [Think outside the container](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/51253), an ...

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md [hunk 20]
```diff
@@ -0,0 +1,97 @@
...
+
+Don't have a ticket yet? Get a free Expo Plus pass for OSCON with the code SPEXPO. Includes events, All the Bots Day, Open Container Day, [and more](http://oreil.ly/1T0euIj)!
```
Comment (kmsholcomb) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: Get a free Expo Hall Plus pass for OSCON with the code SPEXPO. The pass includes events, the “all the bots” day, the “open contribute”  day, ...

**Comment by kmsholcomb** [comment]
OK, @smashwilson. I left a few copy edit in the file. Please let me know if you have any questions. Thanks!

File: _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md (hunk 6 above)
Comment (smashwilson) [inline _posts/2016-05-06-weekly-news-shakespeare-logs-oscon.md]: I'm specifying the excerpt with the [`<!-- more -->` comment](https://jekyllrb.com/docs/posts/#post-excerpts). The blog index is picking it up correctly:

screen shot 2016-05-06 at 3 20 09 pm

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-8615bb5aa6/). :bow:

**Comment by smashwilson** [comment]
Thanks! :100:

Edits complete - now, let's make sure that my markdown is still intact and :shipit:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-966ff7bcbd/).

**Comment by smashwilson** [comment]
There we go.

**Pull request merged**